src/pageContent/Profile/index.tsx

```diff
@@ -0,0 +1,104 @@
+import Text from "src/components/Text";
+import Button from "src/components/Button";
+import Input from "src/components/Input";
+import { useState } from "react";
+
+const ProfileContent: React.FC<Record<string, never>> = () => {
+  const [name, setName] = useState<string>(undefined);
+  const [subject, setSubject] = useState<string>(undefined);
+  const [content, setContent] = useState<string>(undefined);
+
+  const handleSubmit = (event: React.MouseEvent) => {
+    event.preventDefault();
+
+    if (!name) {
+      setName(""); // Lazy way to trigger errror
+      return;
+    }
+    if (!subject) {
+      setSubject(""); // Lazy way to trigger errror
+      return;
+    }
+    if (!content) {
+      setContent(""); // Lazy way to trigger errror
+      return;
+    }
+
+    const body =
+      `Sup Amadeus,\n\n` +
+      `${content}\n\n` +
+      `Thank you.\n\n` +
+      `Best regards,\n\n` +
+      `${name}\n`;
+
+    // Not too sure why the below needs encodeURIComponent. Think because its a URL? 
+    // Oh it doesn't actually send the email just to open the email App, if any. 
+    // What happens if don't have (?)
+    location.href =
+      `mailto:amadeusw6@gmail.com.com` +
+      `&subject=${encodeURIComponent(subject)}` +
+      `&body=${encodeURIComponent(body)}`;
+  };
+
+  return (
+    <>
```

Comment:
What's the purpose of fragment and is fragment really required here ?

Reply:
Good qn gw gtw knp ad fragment d situ lel tyty

Reply:
wtf lmao

Reply:
masi ada fragment ?